src/mercator-viewport.js

```diff
@@ -200,45 +202,7 @@ export default class MercatorViewport {
    *   Per cartographic tradition, lat and lon are specified as degrees.
    */
   unprojectFlat(xyz, scale = this.scale) {
-    return this._unprojectFlat(...arguments);
+    return xyz;
   }
 
-  getMatrices({modelMatrix = null} = {}) {
-    let modelViewProjectionMatrix = this.viewProjectionMatrix;
-    let pixelProjectionMatrix = this.pixelProjectionMatrix;
-    let pixelUnprojectionMatrix = this.pixelUnprojectionMatrix;
-
-    if (modelMatrix) {
```

Comment:
We are dropping modelMatrix support, assuming it was never documented in this module

Reply:
We never had `modelMatrix` support - This method is copied from deck.gl and not called anywhere.